### PR TITLE
Oauth

### PR DESCRIPTION
**Pull request by longarms07**
Demo for Oauth is complete, with back and front end integration. Just need to reconfigure the google secret keys to call env instead of config, the code to do so is present just commented out.

### CODE REVIEW

**Comment by AmiliaM** [comment]
Sorry for the delay. I merged in the CI tests. Please fix the issues that Travis failed on, and I can continue the review :)

File: gradGyde/templates/dash_stud.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+<head>
+    <title>gradGyde Dashboard</title>
+
+    <link rel="stylesheet" type="text/css" href="../static/bootstrap.min.css" />
```
Comment (AmiliaM) [inline gradGyde/templates/dash_stud.html]: There is no point in uploading these to GitHub, it just makes a mess, and there are legal ramifications for distributing code that we do not own. Please use a CDN, for example [Bootstrap's official CDN](https://www.bootstrapcdn.com).

File: gradGyde/static/style.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
+body {
+    background-color: #222222;
+    font-family: "Verdana", Geneva, sans-serif;
+    color: white;
+    font-size: 20px;
+
+}
+
+button {
+  backrgound-color: #aeb4b7;
```
Comment (AmiliaM) [inline gradGyde/static/style.css]: `background-color` typo?

File: gradGyde/templates/dash_stud.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        <li class="nav-item">
+            <a class="nav-link active" id="dashboard-tab" data-toggle="tab" href="stub.html" role="tab" aria-controls="dashboard" aria-selected="true">Dashboard</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link" id="lac-tab" data-toggle="tab" href="stub.html" role="tab" aria-controls="lac" aria-selected="false">LACs</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link" id="explore-tab" data-toggle="tab" href="stub.html" role="tab" aria-controls="explore" aria-selected="false">Explore</a>
+        </li>
+<<<<<<< HEAD
```
Comment (AmiliaM) [inline gradGyde/templates/dash_stud.html]: Some conflict markers were committed here.

File: gradGyde/main.py [hunk 4]
```diff
@@ -1,9 +1,87 @@
...
+def oauth_google():
+    return GOOGLE.authorize(callback=url_for('oauth_google_authorized',
+                                             _external=True))
+
+
+@app.route('/authorized/')
+def oauth_google_authorized():
+    resp = GOOGLE.authorized_response()
+    if resp is None:
+        print('You denied the request to sign in.')
```
[the remark this answers is not in the document]
Comment (longarms07) [reply gradGyde/main.py]: Removed the print statements

**Pull request merged**